src/main/java/de/medizininformatikinitiative/torch/util/ElementCopier.java

```diff
@@ -81,6 +81,7 @@ public void copy(DomainResource src, DomainResource tgt, Attribute attribute) th
             logger.trace("Elements received {}", fhirPath);
             if (elements.isEmpty()) {
                 if (attribute.isMustHave()) {
+                    logger.debug("Must Have Violated Thrown in Copier for {} {}",attribute.getAttributeRef(),ResourceUtils.getPatientId(src));
```

Comment:
@Lucas0T  is here not the information missing which attribute group it is

Reply:
attribute group info is not present at this level

Reply:
Needs a parent comment